api/docs/v2/storefront/index.yaml

```diff
@@ -21,7 +22,7 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
       security:
-        - bearerAuth: []
+        - orderToken: []
```

Comment:
We should keep both options(orderToken and bearerAuth)

Reply:
So, we don't want to let users log in?

Reply:
bearerAuth is used to authenticate user so we can associate order with user.

Reply:
Cart and Checkout should be authorized only the Order token - as for the guest and signed in users.

Reply:
but we need some way to associate logged in user with order